.gitignore

```diff
@@ -52,3 +52,4 @@ GTAGS
 GRTAGS
 GPATH
 GSYMS
+/.vs
```

Comment:
Why add this into .gitignore?
It does not seem to be related to the issue.

Reply:
I agree with @z1ggy-o, this change needs to be removed from your commit.

It is a separate problem. and should not be included in your PR.

Reply:
I removed the addition in line right from Github and it looks like it passed all tests.

I will figure out how to prevent that from happening at the source. Let me know if I need to provide additional edits.